src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -0,0 +1,135 @@
+
+        if (!(brazeSwitch && apiKey)) {
+            resolve({ result: false });
+            return;
+        }
+
+        if (isDigitalSubscriber) {
+            resolve({ result: false });
+            return;
+        }
+
+        Promise.all([asyncBrazeUuid, hasRequiredConsents()]).then(
+            ([brazeUuid, hasGivenConsent]) => {
+                if (!(brazeUuid && hasGivenConsent)) {
+                    resolve({ result: false });
+                } else {
+                    import(
+                        /* webpackChunkName: "braze-web-sdk-core" */ '@braze/web-sdk-core'
+                    )
+                        .then(({ default: appboy }) => {
+                            appboy.initialize(apiKey, {
+                                noCookies: true,
+                                baseUrl: 'https://sdk.fra-01.braze.eu/api/v3',
+                                sessionTimeoutInSeconds: 1,
+                                minimumIntervalBetweenTriggerActionsInSeconds: 0,
+                            });
+
+                            appboy.subscribeToInAppMessage((message) => {
+                                const meta = (message as any).extras;
+
+                                if (meta) {
+                                    resolve({ result: true, meta });
+                                } else {
+                                    resolve({ result: false });
+                                }
+                            });
+
+                            appboy.changeUser(brazeUuid);
+                            appboy.openSession();
+                        })
+                        .catch(() => {
+                            resolve({ result: false });
+                        });
+                }
+            },
+        );
+    });
+
+export const BrazeBanner = ({ meta }: Props) => {
+    const [ExampleComponent, setExampleComponent] = useState<
+        React.FC<BrazeBannerProps>
+    >();
+
+    useEffect(() => {
+        if (meta && meta['test-key']) {
+            // TODO: unify the way we handle sharing these deps (this is
+            // duplicated in SlotBodyEnd). Probably via the automat client
+            // library.
+            window.guardian.automat = {
+                react: React,
+                preact: React,
+                emotionCore,
+                emotionTheming,
+                emotion,
+            };
+
+            import(
+                /* webpackChunkName: "guardian-braze-components" */ '@guardian/braze-components'
+            )
+                .then((module) => {
+                    setExampleComponent(() => module.ExampleComponent);
+                })
+                .catch((error) =>
+                    window.guardian.modules.sentry.reportError(
+                        error,
+                        'braze-banner',
+                    ),
+                );
+        }
+    }, [meta]);
+
+    if (ExampleComponent && meta && meta['test-key']) {
```

Comment:
You might be able to use the funky `meta?.['test-key']` now instead of `meta && meta['test-key']`.

Reply:
Ah yeah for some reason I thought that the linter or something wasn't happy with optional chaining in DCR. Definitely better if I can use it.

Reply:
It turns out there is an issue with optional chaining currently - for some reason Jest isn't happy with it (I get an `unexpected token` error).

Reply:
Ah okay, yeah not worth fighting that really.